versions-plone-5.2-latest.cfg

```diff
@@ -400,4 +401,80 @@ plone.app.standardtiles = 2.3.2
 plone.app.tiles = 3.2.0
 plone.tiles = 2.3.0
 plone.jsonserializer = 0.9.10
+argcomplete = 1.11.1
+argh = 0.26.2
+args = 0.1.0
+asn1crypto = 1.3.0
+cached-property = 1.5.1
+cffi = 1.14.5
+check-manifest = 0.45
+clint = 0.5.1
+cryptography = 3.3.2
+Deprecated = 1.2.10
+distro = 1.5.0
+ecdsa = 0.15
+entrypoints = 0.3
+fancycompleter = 0.9.1
+FormEncode = 1.3.1
+gitdb = 0.6.4
+gitdb2 = 2.0.6
+GitPython = 2.1.15
+html5lib = 1.0.1
+httplib2 = 0.18.1
+keyring = 19.3.0
+launchpadlib = 1.10.13
+lazr.authentication = 0.1.3
+lazr.restfulclient = 0.14.3
+lazr.uri = 1.0.3
+oauth = 1.0.1
+oauthlib = 3.1.0
+pathtools = 0.1.2
+pdbpp = 0.10.2
+progress = 1.5
+prompt-toolkit = 1.0.18
+pycrypto = 2.6.1
+PyGithub = 1.47
+pyrepl = 0.9.0
+pyroma = 2.6.1
+PyYAML = 5.3.1
+SecretStorage = 3.1.2
+smmap = 3.0.1
+smmap2 = 3.0.1
+stdlib-list = 0.6.0
+testresources = 2.0.1
+wadllib = 1.3.3
+watchdog = 1.0.2
+wcwidth = 0.1.9
+wmctrl = 0.3
+wrapt = 1.12.1
+z3c.dependencychecker = 2.7
+zest.pocompile = 1.5.0
+[versions:python27]
+functools32 = 3.2.3.post2
+futures = 3.3.0
+sourcecodegen = 0.6.14
+trollius = 2.2.post1
+docutils = 0.15.2
+twine = 1.15.0
+markdown = 3.1.1
+mock = 3.0.5
+
+# BBB only packages (to be removed with Plone 5.3 or 6.0 and in 5.2 on Python 3.6+
+plone.app.controlpanel                = 4.0.1
+
+# Archetypes
+# part of core, listed separate b/c of planned removal for 6.0 and 5.2 on Python 3
+archetypes.referencebrowserwidget     = 2.5.11
+archetypes.schemaextender             = 3.0.1
```

Comment:
Do we really need this?

Reply:
it is automatically generated...

Reply:
From what / which script? `show-picked-versions = true`?
That would mean that Euphorie pulls in those dependencies.
Or maybe I'm not getting something

Reply:
Nevermind, I would not focus on those pins, really...

Reply:
yeah, fine for me.
I was just wondering where this comes from, as we generally try to avoid redundancy and repeating things that we actually do not need.